extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -7,19 +7,29 @@
  * container for a single list item.
  */
 
+ // Replace with RX.Fragment when support is added in ReactXP major release
+import { Fragment } from 'react';
 import * as assert from 'assert';
 import * as RX from 'reactxp';
 
 export interface VirtualListCellInfo {
     key: string;
 }
 
+export interface VirtualListCellRenderDetails<T extends VirtualListCellInfo> {
+    item: T;
+    selected: boolean;
+    focused: boolean;
+}
+
 export interface VirtualListCellProps<ItemInfo extends VirtualListCellInfo> extends RX.CommonProps {
     // All callbacks should be prebound to optimize performance.
     onLayout?: (itemKey: string, height: number) => void;
     onAnimateStartStop?: (itemKey: string, start: boolean) => void;
-    onCellFocus?: (itemKey: string|undefined) => void;
-    renderItem: (item: ItemInfo, focused: boolean) => JSX.Element | JSX.Element[];
+    onCellFocus?: (itemKey: string | undefined) => void;
+    onItemSelected?: (item: ItemInfo) => void;
```

Comment:
Why is onItemSelected passed the full item but onCellFocus is passed only the key? Seems inconsistent. 

Similarly, I wonder if we should be more consistent with the naming and change onCellFocus to onItemFocus?

Reply:
onCellFocus is a prop only usable by VirtualListView iteself to know when the VirtualListCell is focused
I could rename it & change the signature if you want, but no data besides the key will be used

Reply:
For consistency, I think it's worth changing.